blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
```

Comment:
Perhaps worth also stating the signature of the function i.e. `onSelect()` (or at least I'd been treating it as zero-arg callback -- perhaps worth actually implementing it as such to make sure callers don't accidentally depend on the event being passed through)